### DISCUSSION TITLE
Ability to track different account types for different investment goals

### DISCUSSION THREAD

**Discussion by ups7** [ghostfolio/ghostfolio#2473] [Ideas]
Hello,

I'd like to request a feature to specify an account type/category on the account level, and being able to filter by account type on all screens (Overview, Portfolio etc.). I explain the purpose below.

In most countries, there are different types of investment accounts. I'll use the UK as an example. As a minimum, you can have a SIPP (Self-Invested Pension Plan), ISA (Individual Savings Account) and a GIA (General Investment Account or a regular trading account which you get anywhere).

Each of these accounts serves a different purpose and has different investment objectives:

- **SIPP:** You cannot withdraw anything until a certain age (55) close to retirement, so these are really long term investments that you cannot touch for a long time depending on your age.
- **ISA:** You can deposit up to 20k per year, but all gains are tax free - whether capital gains or dividends. You can withdraw anytime, so it can be useful for medium-term investments.
- **GIA:** Beyond (rapidly diminishing) tax-free allowances, all gains are taxed but there is no limit to how much you can deposit.

Note: You can hold multiple accounts of each type.

As you can see, it is beneficial to track these accounts independently from one another as they have different investment horizons and objectives. I'll not use the SIPP until I'm close to retiring but may use the ISA for children's education or other major expenses. And I may use the GIA in a different way to optimise the annual tax-free allowances.

I believe that most countries have their own variations of such accounts, and UK isn't special.I think that having a dropdown on the account level, and then allowing filtering by each of these account types to see the Overview, Holdings, Summary, Analysis etc. will be very useful in keeping track of your investment objectives.

What do you think?

Thanks,
ups

**Comment by dtslvr** [comment]
For a similar use case, I have created a tag `RETIREMENT_PROVISION` and link it to corresponding activities.
If tags were also available at the account level, you could probably cover your needs completely.

**Reply by ups7** [reply]
Yes, I also thought that tags could meet the requirement. Although I think this is more fundamental and may be worth differentiating from tags which could cause confusion if used at both account and transaction levels?

Also, the Overview screens don't allow filtering by tags so that's something which will still be needed. I use those screens most frequently to check my investments, and currently resort to my spreadsheet to see the account-type level breakdown (I'm having to maintain my spreadsheet too for now).

**Reply by denispol** [reply]
Indeed being able to distinguish at account level would be great. For example with pension funds, the balances are not cash/liquid, and we may not know exactly where they are invested. In Ghostfolio they are just displayed as buying power. Same goes for 3rd pillar in Switzerland, which is invested in clearly visible assets, but may be very hard to track in Ghostfolio (no simple export from VIAC or other providers, except Finpension), hence I just refresh the cash balance, and it's displayed as cash in the net worth

**Reply by dtslvr** [reply]
@denispol For Swiss 3rd pillar accounts with unknown assets I also track the cash balance only, but I have set _Exclude from Analysis_ at account level.

**Comment by jack-ben** [comment]
Same request here.
